src/Components/Form_register/Form_Register.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react'
+import './Form_Reg.css'
+class Form_Register extends React.Component{
+
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            Name: '',
+            Email: '',
+            PhoneNumber:'',
+            Password: '',
+            Avatar: '',
+            Gender: 'Male',
+            Age: 1,
+            Note: '',
+            Favorite: []
+            }
+        }
+
+    onChangeFavorite = (value) => {
+        let New_Favorite = Array.from(this.state.Favorite)
+        if( New_Favorite.indexOf(value) === -1){
+            New_Favorite.push(value)
+            console.log('New_Favorite',New_Favorite)
+            console.log('Add success!' ,value)
+        }else{
+            let index = New_Favorite.findIndex( item => item === value )
+            New_Favorite.splice(index,1)
+            this.setState({Favorite: New_Favorite})
+            console.log('Remove success!' ,value)
+            console.log('New_Favorite',New_Favorite)
+        }
+        this.setState({Favorite: New_Favorite})
+    }
+
+    onChangeGenger = (value) => {
+        this.setState({Gender: value})
+    }
+
+    HandleChangeInput = (event) => {
+        let Value = event.target.value
+        let Name = event.target.name
```

Comment:
```suggestion
    const { name, value } = event.target
```
Viết tất cả các hàm onChange thành 1 hàm thôi,
ko tách ra thành 1 đống hàm change gender, change input, ... như thế này đang bị dublicate code (trùng lặp code quá nhiều)